Assets/Scripts/Nodes/Buddy Nodes/CheckHunger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using UnityEngine;
+using System.Collections;
+using BehaviorTree;
+
+public enum Comparison
+{
+	GreaterThan,
+	EqualTo,
+	LessThan
+}
```

Comment:
Probably should move this out somewhere like BuddyBehaviourUtils just so its not in a random node